### PR TITLE
feat: Add useAppsInMaintenance

### PR DESCRIPTION
**Pull request by flohhhh**
Allows to use it in other projects.

Today we use it in cozy-home.
We need to use it in cozy banks.

### CODE REVIEW

**Comment by flohhhh** [comment]
Problems of `Cannot write file 'client/packages/cozy-client/types/models/note.d.ts' because it would overwrite input file.`

[Trick to resolve ?](https://github.com/cozy/cozy-client/commit/ced357d0e5f87e4fb45d15e05a7827565bfa0fce)

**Comment by Crash--** [comment]
Can you create functional commits? I think you can squash a few of them

**Comment by flohhhh** [comment]
> Can you create functional commits? I think you can squash a few of them

Done

File: packages/cozy-client/src/hooks/useAppsInMaintenance.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react'
+import CozyClient from '../CozyClient'
+import Registry from '../registry'
+
+/**
+ * Returns all apps in maintenance
+ *
+ * @param {CozyClient} client CozyClient instance
+ *
+ * @returns {Array} An array with all apps in maintenance
```
Comment (Crash--) [inline packages/cozy-client/src/hooks/useAppsInMaintenance.jsx:10]: AppsDoctype

**Review by Crash-- — APPROVED** [review]

**Pull request merged**